regression/contracts/history-pointer-replace-03/test.desc

```diff
@@ -4,7 +4,7 @@ main.c
 ^EXIT=(1|64)$
 ^SIGNAL=0$
 ^CONVERSION ERROR$
-error: __CPROVER_old expressions are not allowed in __CPROVER_requires clauses
+error: __CPROVER_old expressions are not allowed
```

Comment:
Not allowed where? Could you improve this error message?

Reply:
The line number is printed on the standard output, but we don't match it in the regex. Should we match the line number here?